plugin/src/main/java/org/opensearch/ml/action/undeploy/TransportUndeployModelAction.java

```diff
@@ -229,7 +229,6 @@ protected MLUndeployModelNodeResponse nodeOperation(MLUndeployModelNodeRequest r
 
     private MLUndeployModelNodeResponse createUndeployModelNodeResponse(MLUndeployModelNodesRequest MLUndeployModelNodesRequest) {
         mlStats.getStat(MLNodeLevelStat.ML_EXECUTING_TASK_COUNT).increment();
-        mlStats.getStat(MLNodeLevelStat.ML_REQUEST_COUNT).increment();
```

Comment:
undeploy model request not counted into request count?

Reply:
we do that here: https://github.com/opensearch-project/ml-commons/blob/2.x/plugin/src/main/java/org/opensearch/ml/model/MLModelManager.java#L1052-L1079